charts/govuk-rails-app/templates/assets-upload-job.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: batch/v1
```

Comment:
Would be good to wrap this in a `if` statement and disable it for apps that don't have assets at `/app/public/assets`.

Though this helm chart is called `govuk-rails-app` it's used by non-Rails Ruby apps and Router (Golang).

Reply:
I wasn't aware of that - talking it through with Chris now to identify things like that, so thanks!

Reply:
done!

Reply:
We also made the script (hopefully) a harmless no-op when `/app/public/assets/$name_of_app` doesn't exist, so it shouldn't break stuff when someone adds an app that doesn't have an assets directory but doesn't disable the assets upload.